From 70217bdbe49280373fd76a453990e733b6f8f9af Mon Sep 17 00:00:00 2001
From: Wolfgang Grandegger <wg@aries-embedded.de>
Date: Wed, 24 May 2023 09:42:18 +0200
Subject: [PATCH 7/7] configs: socfpga_mcvevk: generate random mac address if
 ethaddr is unset

This is necessary to make the Ethernet interface function even when
the ethaddr variable for that interface is unset.

Signed-off-by: Wolfgang Grandegger <wg@aries-embedded.de>
---
 configs/socfpga_mcvevk_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/socfpga_mcvevk_defconfig b/configs/socfpga_mcvevk_defconfig
index 8c939614f1..6f5856a94f 100644
--- a/configs/socfpga_mcvevk_defconfig
+++ b/configs/socfpga_mcvevk_defconfig
@@ -53,6 +53,7 @@ CONFIG_SYS_RELOC_GD_ENV_ADDR=y
 CONFIG_USE_BOOTFILE=y
 CONFIG_BOOTFILE="fitImage"
 CONFIG_VERSION_VARIABLE=y
+CONFIG_NET_RANDOM_ETHADDR=y
 CONFIG_DFU_MMC=y
 CONFIG_SYS_DFU_DATA_BUF_SIZE=0x1000000
 CONFIG_DWAPB_GPIO=y
-- 
2.34.1

